registry/default/akasha-ui/copy-to-clipboard.tsx

```diff
@@ -0,0 +1,71 @@
+"use client";
+
+import * as React from "react";
+
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "@/registry/default/ui/tooltip";
+
+interface CopyToClipboardProps {
+  textToCopy: string;
+  initialTooltipText?: string;
```

Comment:
Can we rename this ? Something like ctaText(call to action text) without including the word tooltip ... the international implementation can change but keeping the prop name independent from internal implementation is good